templates/search/catalogue_search.html

```diff
@@ -31,48 +43,63 @@
+                    </div>
                 </div>
-                <div class="tna-column tna-column--width-2-3 tna-column--full-medium tna-column--full-small tna-column--full-tiny tna-!--margin-top-s">
-                    {% include './blocks/search_results.html' %}
+            {% endif %}
 
-                    {% if page.paginator.count %}
-                        {% include './blocks/search_pagination.html' %}
-                    {% endif %}
+            {% if buckets.current.result_count > 0 or form.errors %}
+                <div class="tna-container tna-!--margin-top-s" data-id="catalogue-search-grid">
+                    <div class="tna-column tna-column--width-1-3 tna-column--full-medium tna-column--full-small tna-column--full-tiny tna-!--margin-top-s" data-id="catalogue-search-sidebar">
+                        {% include './blocks/search_filters.html' %}
+                    </div>
+                    <div class="tna-column tna-column--width-2-3 tna-column--full-medium tna-column--full-small tna-column--full-tiny tna-!--margin-top-s">
+                        {% include './blocks/search_results.html' %}
 
+                        {% if page.paginator.count %}
+                            {% include './blocks/search_pagination.html' %}
+                        {% endif %}
+
+                    </div>
                 </div>
-            </div>
-            </div>
-        {% else %}
-            {# No results were found when searching with the 'q' param value for all buckets ... #}
-            {# ... or for only current bucket while some other buckets have results #}
-            <div class="tna-container">
-                <div class="tna-column tna-column--full">
-                    {% include './blocks/no_results.html' %}
                 </div>
+            {% else %}
+                {# No results were found when searching with the 'q' param value for all buckets ... #}
+                {# ... or for only current bucket while some other buckets have results #}
+                <div class="tna-container">
+                    <div class="tna-column tna-column--full">
+                        {% include './blocks/no_results.html' %}
+                    </div>
+                </div>
+            {% endif %}
+
+            <div>
+                {{ form.group.errors }}
+                {{ form.group.as_hidden }}
             </div>
         {% endif %}
 
-        <div>
-            {{ form.group.errors }}
-            {{ form.group.as_hidden }}
-        </div>
-
     {% endwith  %}
 
 {% endblock %}
 
 {% block extra_js %}
-    <script src="{% static 'scripts/catalogue_search.js' %}"></script>
+    {% if form.views.value != views.MAP.value %}
+        <script src="{% static 'scripts/catalogue_search.js' %}"></script>
+    {% endif %}
+
+    {% if form.views.value == views.MAP.value %}
+        <script src="{% static 'scripts/search_results_map.js' %}"></script>
+    {% endif %}
```

Comment:
```suggestion
    {% if form.views.value == views.MAP.value %}
        <script src="{% static 'scripts/search_results_map.js' %}"></script>
    {% else %}
        <script src="{% static 'scripts/catalogue_search.js' %}"></script>
    {% endif %}
```
Why not use an else here?